packages/farmhash/farmhash.0.4/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "github@ahrefs.com"
+authors: "Ahrefs <github@ahrefs.com>"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/ocaml-farmhash"
+dev-repo: "git+https://github.com/ahrefs/ocaml-farmhash.git"
+bug-reports: "https://github.com/ahrefs/ocaml-farmhash/issues"
+tags: ["org:ahrefs" "clib:farmhash"]
+build: [ "dune" "build" "-p" name "-j" jobs ]
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "dune" {>= "2.0"}
+  "conf-c++"
+]
+depexts: [
+  ["libfarmhash-dev"] {os-family = "debian"}
+]
```

Comment:
I could not find rpms or the packages for alpine or arch. They can be added later if somebody finds them